crates/aptos-telemetry-service/src/prometheus_push_metrics.rs

```diff
@@ -24,16 +24,7 @@ pub async fn handle_metrics_ingest(
     claims: Claims,
     metrics_body: Bytes,
 ) -> anyhow::Result<impl Reply, Rejection> {
-    let extra_labels = vec![
-        format!("peer_id={}", claims.peer_id),
```

Comment:
I removed `peer_id` as label override from the service since `peer_id` is actually already used by a bunch of our network metrics and in that context means the peer_id of _other_ nodes that the node which emits the metrics is connected to. setting peer_id in the telemetry service would override/drop this information.
We already have the peer_id which refers to itself in the kubernetes_pod_name, so we're good.

Reply:
should we trust the `peer_id` sent by the node in the metrics?

Reply:
I guess we have to. I mean the peer_id in the metrics just literally refers to an actual "peer".
The way how we set it previously it was essential more like a "self_id". We could add a new label "self_peer_id" or something that denotes the validated self peer id but it would be just serve the same purpose as the pod_name label we set currently.
Let's have a quick discussion on slack about this.